bin/clean

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 hash mutagen-compose || exit 1
 
-mutagen-compose down --volumes --rmi local
-rm -rf reticulum/{_build,deps}
+basedir=$(dirname $0)
+mutagen-compose -f $basedir/../docker-compose.yml down --volumes --rmi local
+rm -rf $basedir/../services/reticulum/deps
+$basedir/init
```

Comment:
It think it would be a bit surprising to me for the `clean` script to also do an `init`. I would probably leave those as separate operations for the user to decide. Not a big deal though.

Reply:
I was torn on this one. `bin/clean` is destructive—it deletes application artifacts—but not completely so—it does not delete service source code. At first I added a note to the README that `bin/up` will not succeed after a clean without a `bin/init` invocation.

Reply:
If you could detect a failure in `bin/up`, maybe you could print a reminder to `bin/init`

Reply:
Fine to merge as is though. I don't think this is something people will run into often.

Reply:
Maybe `reset` is a more fitting name than `clean`?

Reply:
Yeah, `reset` would definitely be a better name.